09_elements.html

```diff
@@ -97,7 +97,227 @@
     </section>
   </section>
+
+  <section>
+   <h2 id=element-visibility>Element Visibility</h2>
+
+   <p>The visibility of a <a
+    href=http://w3c.github.io/dom/#element>Document element</a>
+    is guided by what is perceptually visible to the human eye.
+    An element's visibility does in this context thus not relate
+    in any way to the <code><a
+    href=http://dev.w3.org/csswg/css-box/#visibility>visibility</a></code>
+    or <code><a
+    href=http://dev.w3.org/csswg/css-box/#display0>display</a></code>
+    style properties [[!CSS3BOX]].
+
+   <p>The approach used by WebDriver to ascertain an element's
+    visibility is based on crude approximations about the element's
+    nature and relationship in the tree.  An element is in general
+    to be considered visible if any part of it is drawn on the
+    canvas within the bounderies of the viewport.
+
+   <p>An element's <dfn>element visibility</dfn>
+    is a boolean state where true signifies visibility and false
+    invisibility.  To compute the state on <var>element</var>:
+
+   <ol>
+    <li>If the attribute <code>hidden</code> is set, return false.
+
+    <li>If the computed value of the <code>display</code> style
+     property is "none", return false.
+
+     <!-- TODO(ato): Not really sure what this means, needs review: -->
+    <li>If it has a [[!CSS3-2D-TRANSFORMS]] or [[!CSS3-3D-TRANSFORMS]]
+     style property that gives a negative X or Y coordinates to
+     the canvas, return false.
+
+    <li>If <var>element</var> is the document's root element, that
+     is <code>document.documentElement</code>:
+
+     <ol>
+      <li>If the computed value of the <code>background-color</code>
+       property is "transparent", run these substeps:
+
+       <ol>
+        <li>If <var>element</var> is an HTML <code>HTML</code>
+         element [[!HTML51]], and the computed value of the
+         <code>background-color</code> style property of the first
+         <code>BODY</code> element descendant of the element in
+         <a
+         href=http://www.whatwg.org/specs/web-apps/current-work/multipage/infrastructure.html#tree-order>tree
+         order</a>, relative to that element, is also "transparent",
+         return false.
+
+        <li>Otherwise return false.
+       </ol>
+     </ol>
+
+    <li>If <var>element</var> is an <code>option</code> or
+     <code>optgroup</code> element:
+
+     <ol>
+      <li>For each immediate ancestral element <var>parent</var>
+      of <var>element</var>:
+
+      <ol>
+       <li>If <var>parent</var> is a <code>select</code> element,
+        return the result of applying the <a>element visibility</a>
+        algorithm on it if its result is false.
```

Comment:
The end of the sentence loses me. Can you be more explicit instead of using `it` and `its`